proxy/http/Http1ClientSession.cc

```diff
@@ -345,11 +352,21 @@ Http1ClientSession::state_keep_alive(int event, void *data)
 {
   // Route the event.  It is either for vc or
   //  the origin server slave vc
-  if (data && data == slave_ka_vio) {
-    return state_slave_keep_alive(event, data);
-  } else {
-    ink_assert(data && data == ka_vio);
-    ink_assert(read_state == HCS_KEEP_ALIVE);
+  if (data) {
```

Comment:
Would this possibly read nicer as a switch statement?

Reply:
Cannot have variables as case elements.